language-nocache.html

```diff
@@ -4,16 +4,10 @@
     <div class="row">
         <div class="page-header">
             <h1>Browse by Language</h1>
-            <p> Below is a list of languages. Select one of the languages to navigate to a list of
-                news reports on Wilde's trials which were published in that language. The number
-                next to each language indicates the number of reports published in that language.
-            </p>
-        </div>
-    </div>
-    <div class="row">
-      <div class='col-md-12'>
           <p> Select one of the languages below to discover news reports on Wilde's trials published in that language. The number next to each language indicates the number of reports published in that language. </p> 
             <div data-template="app:browse-language"/>
-        </div>
```

Comment:
This would put everything into the page header, above the horizontal line. I don't think that's what you want. This would put Browse by Language in the header and the paragraph and list of languages below the horizontal line.

```xml
<?xml version="1.0" encoding="UTF-8"?>
<div xmlns="http://www.w3.org/1999/xhtml" data-template="templates:surround"
    data-template-with="templates/page.html" data-template-at="content">
    <div class="row">
        <div class="page-header">
            <h1>Browse by Language</h1>
        </div>
        <p> Select one of the languages below to discover news reports on Wilde's trials published
            in that language. The number next to each language indicates the number of reports
            published in that language. </p>
        <div data-template="app:browse-language"/>
    </div>
</div>
```